blues/simulation.py

```diff
@@ -224,6 +224,10 @@ def chooseMove(self):
 
     def simulateNCMC(self, verbose=False):
```

Comment:
I think we should add a `write_ncmc` parameter here to make it a bit more obvious we can pass that option here. Then everything inside could just be pointing to `write_ncmc`